performance/pom.xml

```diff
@@ -22,6 +22,7 @@
     <properties>
         <!-- Disable enforcer as we are using banned dependencies for old versions -->
         <enforcer.skip>true</enforcer.skip>
+        <add.bv2.sources.skip>true</add.bv2.sources.skip>
```

Comment:
Maybe a comment would help here... ? Not sure what's the purpose of this.

Reply:
Yeah will add something. 
The thing is that the helper plugin was used to add the sources, and since I've used the same plugin to parse the version in the root pom, now adding the sources happens all the time 🙈 unless we skip it 

And thinking more about it ... we probably should just clean up the perf module. See all those other profiles, they are for `javax` packages so none of them really work as perf test in the main/java are for `jakarta`...
sooooooooooo we probably should have to "optional" source directories one for javax one for jakarta with the same tests in them and then include the dir depending on the spec we are targeting ....

Reply:
I've created a https://hibernate.atlassian.net/browse/HV-2010 so that I stop adding more and more to this PR 🙈 
and a https://hibernate.atlassian.net/browse/HV-2009 for jakarta rename (part of this PR)